auth/src/args.rs

```diff
@@ -37,9 +38,9 @@ pub struct StartArgs {
 
 #[derive(clap::Args, Debug, Clone)]
 pub struct InitArgs {
-    /// Name of initial account to create
+    /// User id of account to create
     #[arg(long)]
-    pub name: String,
+    pub user_id: UserId,
```

Comment:
Wait, the init should still take in the account name right?

Reply:
The account name should be seen as the auth0 name, so the custom names that some accounts use should ideally be in user_id as `user_mycustomname` (prefix being just a convention and not a restriction)

Reply:
Ok, sounds good :+1: